terraform/aws/security/aws-transfer-server-is-public.yaml

```diff
@@ -0,0 +1,27 @@
+rules:
+- id: aws-transfer-server-is-public
+  patterns:
+  - pattern-either: 
+    - pattern: |
+        resource "aws_transfer_server" $ANYTHING {
+          ...
+        }
+    - pattern: |

```

Comment:
Based on the test case, is this supposed to be `pattern-not-inside`?

Reply:
That's true, it should be `pattern-not-inside` to look for `endpoint_type = "VPC"`. Will fix